From f90c58f219e859a6a0618de80b38e4a8c0e6e4f3 Mon Sep 17 00:00:00 2001
From: Andrey Sorokin <Andrey.I.Sorokin@gmail.com>
Date: Thu, 26 May 2022 18:01:10 +0400
Subject: [PATCH] Removed Dill from inference.pyx

---
 pyross/inference.pyx | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyross/inference.pyx b/pyross/inference.pyx
index 87991bb..03ab77d 100644
--- a/pyross/inference.pyx
+++ b/pyross/inference.pyx
@@ -17,7 +17,6 @@ import time, sympy
 from sympy import MutableDenseNDimArray as Array
 from sympy import Inverse, tensorcontraction, tensorproduct, permutedims
 from scipy import interpolate
-import dill
 import hashlib
 
 try:
-- 
2.36.1

